ipatool

```diff
@@ -8,6 +8,10 @@ Usage:
   ipatool [options] [-v...] push [--branch=BRANCH...] [--reviewer=NAME...] [--] [PATCH ...]
   ipatool [options] [-v...] start-review [-f] [--am] [--ticket=NUMBER...] [--] [PATCH ...]
   ipatool [options] [-v...] am [--] [PATCH ...]
+  ipatool [options] [-v...] pr-ack --id=NUMBER --comment=TEXT
+  ipatool [options] [-v...] pr-list [--state=(open|closed|all)] [--label=NAME]
+  ipatool [options] [-v...] pr-push --id=NUMBER [--branch=BRANCH...] [--reviewer=NAME...]
+  ipatool [options] [-v...] pr-reject --id=NUMBER --comment=TEXT
```

Comment:
IMO according the design, PR ID is supposed to be argument not option
ipatool pr-push 11
not ipatool pr-push --id=11